shared/fixes/bash/display_login_attempts.sh

```diff
@@ -1,14 +1,10 @@
 # platform = Red Hat Enterprise Linux 7
 
-if ! `grep -q ^[^#].*pam_succeed_if.*showfailed /etc/pam.d/postlogin` ; then
-  if ! grep `^session.*pam_succeed_if.so /etc/pam.d/postlogin` ; then
-    echo "session     [default=1]   pam_lastlog.so nowtmp showfailed" >> /etc/pam.d/postlogin
-    echo "session     optional      pam_lastlog.so silent noupdate showfailed" >> /etc/pam.d/postlogin
-  else
-    sed -i --follow-symlinks '/^session.*pam_succeed_if.so/a session\t    optional\t  pam_lastlog.so silent noupdate showfailed' /etc/pam.d/postlogin
-    sed -i --follow-symlinks '/^session.*pam_succeed_if.so/a session\t    [default=1]\t  pam_lastlog.so nowtmp showfailed' /etc/pam.d/postlogin
-  fi
+if ! `grep -q "^session.*pam_lastlog.so" /etc/pam.d/postlogin` ; then
```

Comment:
I suggest getting rid of the negation - you can remove the negating `!` and swap the `then` and `else` bodies.
Next, using `$(...)` instead of `` `...` `` is considered a better practice.